CHANGES.md

```diff
@@ -21,6 +21,7 @@
 - Allow the dry-run mode to continue even after some API call's response were expected by using placeholder values (#262, @gpetiot)
 - Build and run tests for all selected packages when checking distribution tarball
   (#266, @NathanReb)
+- Sanitize git branch names (#271, @gpetiot)
```

Comment:
When adding changelog entry, try to think about users. This one isn't very helpful here. I'd suggest something like:
```suggestion
- Fix a bug that prevented release of a package whose version number contains invalid characters for a git branch by sanitzing the local opam repo branch name. (#271, @gpetiot)
```